lib/file_transfer_agent/gcs_util.js

```diff
@@ -37,15 +41,24 @@ function GCSLocation(bucketName, path) {
 /**
  * Creates an GCS utility object.
  *
- * @param {module} httpclient
- * @param {module} filestream
+ * @param {module} httpClient
+ * @param {module} fileStream
  *
  * @returns {Object}
  * @constructor
  */
-function GCSUtil(httpclient, filestream) {
-  const axios = typeof httpclient !== 'undefined' ? httpclient : require('axios');
-  const fs = typeof filestream !== 'undefined' ? filestream : require('fs');
+function GCSUtil(connectionConfig, httpClient, fileStream) {
```

Comment:
@sfc-gh-pmotacki do we consider it as a breaking change?

Reply:
According to information in the release notes for version 2.0.0, whether the API isn't included in the typing file, we treated it as internal.